src/index.ts

```
@@ -0,0 +1,369 @@
+  }
+
+  const disconnects: Record<number, string> = {};
+  let disconnectCount = 0;
+  let unexpectedCount = 0;
+
+  cluster.on('disconnect', function(worker) {
+    const disableRefork = getDisableRefork(worker);
+    const log = console[ disableRefork ? 'info' : 'error'];
+    disconnectCount++;
+    const isDead = worker.isDead && worker.isDead();
+    const propertyName = worker.hasOwnProperty('exitedAfterDisconnect') ? 'exitedAfterDisconnect' : 'suicide';
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)